### DISCUSSION TITLE
On running Northstar dedicated server on ARM Linux systems

### DISCUSSION THREAD

**Discussion by wisheth** [pg9182/northstar-dedicated#46] [Support]
I'm currently looking to host a Northstar server on an Oracle Cloud Ampere machine. The Docker instance managed by this distro is recommended by the Northstar wiki for running a dedicated server in Linux; however, the image is made for amd64 and does not work on ARM systems from my attempts. Is there a way to get this image working on ARM Linux, and if not, (how) would I be able to build the files myself on an ARM system?

**Comment by Kuratius** [comment]
https://github.com/ptitSeb/box64/issues/210?notification_referrer_id=NT_kwDOAtSDLbMyOTUzNzM4MTY5OjQ3NDgxNjQ1
